txtl-liposome_thin-film.md

```diff
@@ -0,0 +1,109 @@
+
+### Step 1: Prepare DOPC films (2.5E-4 mols DOPC/vial)
+
+1. Determine the volume and molarity DOPC vesicles you want to create
+   ex: 1 mL of 20 mM DOPC vesicles = 2.5E-5 mols of DOPC. The stock of
+   DOPC is 25 mg/mL. To make giant, nice looking vesicles, it’s best
+   to prepare a more dilute sample of vesicles (like 200 uM). For
+   small unilamellar vesicles, you can work with much higher
+   concentrations of lipids (we can go up to 130 mM for DOPC)
+
+2. Determine the dye and molar % of the dye : DOPC you would like
+   ex. for 0.1 mol% rhodamine DHPE :
+
+3. 2.5e-6 moles DOPC×(0.1 moles Rhod Liss PE)/(100 moles
+   DOPC)×L/(moles Rhod in stock)=volume Rhod Liss PE stock to add
+
+4. Add 2e-5 mols DOPC and appropriate amount of Rhodamine PE. Transfer
+   with a Hamilton (glass) syringe into a glass vial since you are
+   working with organic solvents. Cap and vortex briefly to ensure the
+   two components are well mixed.
+
+5. Remove cap and allow the chloroform to evaporate away in the hood.
+
+6. Place the vial into a vacuum chamber for > 1 hrs or leave overnight
+   in order to remove all residual traces of solvent. *Removal of
+   residual chloroform from the lipid film is very important for high
+   quality vesicles
+
+7. Cap and store vials in freezer until ready to be used and hydrate
+   with 1 mL of aqueous hydration buffer when you are ready to prepare
+   vesicles.
+
+### Step 2: Hydrate films and extrude vesicles
+
+1. Add 1 mL of PBS to each vial, place in 60°C oven for 1 hr, then
+   vortex 10 seconds. Lipids should easily assemble. If clumps or
+   aggregates persist after several vortexing attempts, this is often
+   a sign something went wrong in vesicle assembly. The heating step
+   is not necessary and can be excluded, but helps when other membrane
+   components are involved like diblock copolymers.
+
+2. Extrude films at room temperature through 100 nm polycarbonate
+   membranes using a mini extruder and heating block described on the
+   [Avanti
+   website](https://avantilipids.com/divisions/equipment-products/mini-extruder-assembly-instructions)
+   follow the procedure described
+   [here](https://avantilipids.com/divisions/equipment-products/mini-extruder-extrusion-technique)
+   (7 passes through membrane is fine, 9-11 is better, more than that
+   is unnecessary according to my post extrusion analysis of vesicle
+   size distribution on DLS.) Always perform an odd number of passes
+   through the membrane so no vesicles that have never passed through
+   the membrane end up in the final sample.
+
+### Step 3: Purify vesicles if hydration buffer contained a dye or solute (ex. calcein)
+
+1. Set up columns containing 6 mLs of Sepharose 4B (size exclusion
+   column media). Purification should be conducted on the same day as
+   a subsequent assay.
```

Comment:
Probably it should be added how to make the columns (??), which buffer to use (PBS), and how to detect the fractions where vesicles are. I can write something but I am not sure if I can, of if you prefere to do it.

Reply:
@p-stano: Feel free to write something as a comment and I can integrated it.  Alternatively you can fork a copy of my repository and then send me a pull-request (which will eventually show up here).

Reply:
@murrayrm: as I already have a fork copy of the repository, is there a method to synchronize the two? Or shall I create another fork copy? (I see the power of github, but at the beginning it is not easy to understand the best way to do things) - thank you

Reply:
Since you want to make changes on top of my changes, the easiest thing to do is to fork *my* repository (you probably have a fork of the main repository), make your changes, then send me a request to merge your changes into mine (which I can comment on, etc).  The only complication is that we are doing pull requests on pull requests, so it can get confusing (though GitHub will keep track of it).

If you want to update a current fork to the latest changes, you can either do this from the command line (just do "git pull upstream master") or you can do it via the GitHub interface by generating a pull request from the newer repository back into your repository.  (This is one place where it is better to use the command line, but that requires installing git locally...).

Reply:
It seems too complicated. If I have suggestions, I can do later. Thank you.

Reply:
I was thinking about size-exclusion chromatography. Maybe the correct way is to have a separated protocol only for this (it can be useful in several projects). 

Note. This recalls me your comment about "languages". In this case, recalling another technique in a protocol is analogous to recalling a 'procedure' -- thinking in terms of algorithms. Are there examples where these 'languages' have been already applied to common used experiments? I am a bit interested, as I have friends in the computer science that can be also interested.

Reply:
@PaolaTorre or @aaronengelhart: Can one of you suggest a change to address @p-stano's comment?  I can implement once I know what to change.

Reply:
@murrayrm I agree with @p-stano that these details about the chromatography column needs to be included, but I don/t know if everyone purchased this equipment.

Reply:
We typically use http://www.bio-rad.com/en-us/sku/7311553-poly-prep-chromatography-columns-pkg-1-000?ID=7311553 with about 1mL of sepharose.